feature-group-definitions/font-palette-animation.dist.yml

```diff
@@ -0,0 +1,14 @@
+# Generated from: font-palette-animation.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: font-palette animation
+spec: https://drafts.csswg.org/css-fonts-4/#font-palette-prop
+usage_stats: https://chromestatus.com/metrics/css/timeline/animated/709
+status:
+  baseline: false
+  support:
+    chrome: "121"
+    chrome_android: "121"
+    edge: "121"
+compat_features:
+  - css.properties.font-palette.palette-mix_function
```

Comment:
Hold up, are we using the right key? Should this be `css.properties.font-palette.animation_computed`?

Reply:
The `palette-mix()` function can in principle be used without animations, but the only reason it was implemented was for animations. So I think it makes sense in this entry. But I'll also add `css.properties.font-palette.animation_computed` in a separate PR, to be landed after this.